### PR TITLE
CSV to json

### PR DESCRIPTION
**Pull request by emilyb7**
Like #157  but the other way... 

Sorry if this is horrible. It works, I promise.

related #137

### CODE REVIEW

File: lib/parse_csv_json.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const getRowKey = row => row[0];
+
+const getKeyHeading = row => getRowKey(row).split('.')[0];
+
+const getKeyName = row => getRowKey(row).split('.')[1];
+
+const makePair = (row, localeIndex) => _.object([ [ getKeyName(row), getLocaleStr(row, localeIndex), ], ]);
+
+const transformGroup = (group, localeIndex) => group.map(row => makePair(row, localeIndex)).reduce((a,b) => Object.assign({}, a, b), {});
```
Comment (des-des) [inline lib/parse_csv_json.js:25]: new lines:
```js
const transformGroup = (group, localeIndex) => group
  .map(row => makePair(row, localeIndex))
  .reduce((a,b) => Object.assign({}, a, b), {});
```

**Review by des-des — CHANGES_REQUESTED** [review]
Beautiful!

**Comment by des-des** [comment]
@emilyb7 I expect this is quite fragile, but the code is nice.

ie I expect there are cases where the newline regex will fail (this is one `\\n`) but have not gone into detail ask if you want me to go through and think hard about edge cases ..

**Comment by emilyb7** [comment]
@des-des thanks : ) I think it's probs ok for our needs for the time being!

**Review by des-des — APPROVED** [review]

**Pull request merged**